sdk/core/azure-core/src/main/java/com/azure/core/http/policy/AzureSasCredentialPolicy.java

```diff
@@ -74,4 +74,8 @@ public AzureSasCredentialPolicy(AzureSasCredential credential, boolean requireHt
 
         return next.process();
     }
+
+    public String getSasSignature() {
```

Comment:
nit: Missing Javadocs

Reply:
Should we call this method just `getSignature()`? Sas is implied from the policy name.